packages/drop/src/countdown/provider.tsx

```diff
@@ -46,7 +47,7 @@ const CountdownProvider = ({
       console.warn("startDate must be before endDate. Please check your start and end dates.")
     }
 
-    return create<CountdownContext>((set) => {
+    return createWithEqualityFn<CountdownContext>((set) => {
```

Comment:
is this `createWithEqualityFn` needed? could we just use `create`? in the future, [this function may be deprecated](https://github.com/pmndrs/zustand/discussions/1937).

Reply:
yes, we could use `create`. changed it